subworkflows/local/map_alignment.nf

```diff
@@ -2,22 +2,31 @@
  * Perform the quantification of the samples when the parameter --skip_quantification is not provided
```

Comment:
Overall you want to have the quantification step since this is the aspect that aligns the spectra among the different runs. Making the outcome of the data more accurate as well. However, for some use cases you do not want to have the quantification (e.g. when the samples are not that well or if the output does not require quantification). Still, our quantification is the default, since this results in the expected outcome.